services/functions/linkdex.ts

```diff
@@ -0,0 +1,81 @@
+import { APIGatewayProxyHandlerV2, APIGatewayProxyStructuredResultV2 } from "aws-lambda"
+import { S3Client, GetObjectCommand, HeadObjectCommand, paginateListObjectsV2 } from "@aws-sdk/client-s3"
+import { CarBlockIterator } from '@ipld/car'
+import { LinkIndexer } from 'linkdex'
+
+export const handler: APIGatewayProxyHandlerV2 = async (event) => {
+  const key = event.queryStringParameters?.key ?? ''
```

Comment:
Accept multiple `key`? In uploadsV2 CAR parts are unlikely to be in the same directory and so we should be able to ask if a DAG is complete in the _given_ CARs.

Reply:
Yes! But not just yet. I think this will change pretty significantly to be useful for uploads v2.